ee/clickhouse/queries/trends/lifecycle.py

```diff
@@ -87,7 +87,10 @@ def get_query(self):
         date_query, date_params = self._get_date_filter()
         self.params.update(date_params)
 
-        prop_query, prop_params = self._get_prop_groups(self._filter.property_groups)
+        if self._filter.entities and self._filter.entities[0]:
+            prop_query, prop_params = self._get_prop_groups(self._filter.entities[0].property_groups)
+        else:
+            prop_query, prop_params = self._get_prop_groups(self._filter.property_groups)
```

Comment:
Went another step back to figure out how things were supposed to be, since I'm confused how global filters make it in here when they're not shown on the UI.

Turns out this stopped working 3 months ago, and before that, the behaviour was to combine both entity and global filter properties. 

https://github.com/PostHog/posthog/pull/8021/files#diff-4197f769624d6d6541e4d943a4f0e32512f8ade8bce4d83be0aee71238b1437f

I don't like this, since it means if you have a trend with a browser=android mobile filter, and you switch to lifecycle, this filter still applies, while there's zero indication in the UI about this filter.

Thus, suggesting removing global filtering entirely for this PR (or cc: @EDsCODE - adding filters to lifecycle UI, or updating cleanFilters to remove global filters when moving to lifecycle)

```suggestion

```